packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1684,4 +1736,15 @@ export class ColorPicker
   private getAdjustedScopePosition(left: number, top: number): [number, number] {
     return [left - SCOPE_SIZE / 2, top - SCOPE_SIZE / 2];
   }
+
+  private handleColorPickerBlur = (): void => {
+    this.el.blur();
+  };
+
+  private clickHandler = (): void => {
+    if (this.disabled) {
+      return;
+    }
+    this.handleColorPickerBlur();
```

Comment:
question: every time the internal container of the component is clicked, the component is blurred?

It seems a little odd to be calling blur on every click of the component, even if the input is clicked.

Reply:
After reviewing with Franco yesterday, I'm looking for an alternate solution to this. The goal is to be able to blur the input fields without having to click outside of the component.